### PR TITLE
Ruby on rails training

### PR DESCRIPTION
**Pull request by sarmad90**

### CODE REVIEW

File: ruby_on_rails/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  1. [CodeCademy - Learn Ruby on Rails](https://www.codecademy.com/learn/learn-rails)
+
+#### Paid Courses:
+  1. [Udemy - The Complete Ruby on Rails Developer Course](https://www.udemy.com/the-complete-ruby-on-rails-developer-course/)
+  1. [Coursera - Ruby on Rails Web Development Specialization ](https://www.coursera.org/specializations/ruby-on-rails)
+  1. [Lynda - Ruby on Rails Training and Tutorials](https://www.lynda.com/Ruby-on-Rails-training-tutorials/304-0.html)
+  1. [Team Tree House - Rails Development](https://teamtreehouse.com/tracks/rails-development)
+  1. [ONE MONTH RAILS](https://onemonth.com/courses/rails)
+
+#### Tutorials I Have Written:
```
Comment (mohuk) [inline ruby_on_rails/README.md]: `Tutorials I have written` doesn't make sense for a training guide under `10Pearls` name. You can simple move these under helpful links/resources section.

**Comment by sarmad90** [comment]
@mohuk check now

**Comment by mohuk** [comment]
LGTM

**Comment by mohuk** [comment]
took a couple of years to review 🤣

**Comment by sarmad90** [comment]
Haha. No worries.

Sent from my iPhone

On 23-Apr-2019, at 3:29 PM, Muhammad Umair Khan <notifications@github.com<mailto:notifications@github.com>> wrote:


took a couple of years to review 🤣

**Pull request merged**